test/cpuinfo_x86_test.cc

```diff
@@ -335,21 +335,29 @@ TEST_F(CpuidX86Test, Nehalem) {
       PF_XMMI64_INSTRUCTIONS_AVAILABLE);
   g_fake_cpu->SetWindowsIsProcessorFeaturePresent(
       PF_SSE3_INSTRUCTIONS_AVAILABLE);
-#endif  // CPU_FEATURES_OS_WINDOWS
-#if defined(CPU_FEATURES_OS_DARWIN)
+#elif defined(CPU_FEATURES_OS_DARWIN)
   g_fake_cpu->SetDarwinSysCtlByName("hw.optional.sse");
   g_fake_cpu->SetDarwinSysCtlByName("hw.optional.sse2");
   g_fake_cpu->SetDarwinSysCtlByName("hw.optional.sse3");
   g_fake_cpu->SetDarwinSysCtlByName("hw.optional.supplementalsse3");
   g_fake_cpu->SetDarwinSysCtlByName("hw.optional.sse4_1");
   g_fake_cpu->SetDarwinSysCtlByName("hw.optional.sse4_2");
-#endif  // CPU_FEATURES_OS_DARWIN
-#if defined(CPU_FEATURES_OS_LINUX_OR_ANDROID)
+#elif defined(CPU_FEATURES_OS_FREEBSD)
+  auto& fs = GetEmptyFilesystem();
+  fs.CreateFile("/var/run/dmesg.boot", R"(
+  ---<<BOOT>>---
```

Comment:
Even though this will be moot when the branch goes EOL at the end of September, FreeBSD 11 does not have this line in `dmesg.boot`.

Reply:
Ack. It was present in a dump of a FreeBSD 12.2-RELEASE r366954 GENERIC amd64
```
---<<BOOT>>---
Copyright (c) 1992-2020 The FreeBSD Project.
Copyright (c) 1979, 1980, 1983, 1986, 1988, 1989, 1991, 1992, 1993, 1994
	The Regents of the University of California. All rights reserved.
FreeBSD is a registered trademark of The FreeBSD Foundation.
FreeBSD 12.2-RELEASE r366954 GENERIC amd64
FreeBSD clang version 10.0.1 (git@github.com:llvm/llvm-project.git llvmorg-10.0.1-0-gef32c611aa2)
VT(vga): text 80x25
module vtnet already present!
CPU: Intel(R) Core(TM) i7-10710U CPU @ 1.10GHz (1607.96-MHz K8-class CPU)
  Origin="GenuineIntel"  Id=0xa0660  Family=0x6  Model=0xa6  Stepping=0
  Features=0x1783fbff<FPU,VME,DE,PSE,TSC,MSR,PAE,MCE,CX8,APIC,SEP,MTRR,PGE,MCA,CMOV,PAT,PSE36,MMX,FXSR,SSE,SSE2,HTT>
  Features2=0x5eda2203<SSE3,PCLMULQDQ,SSSE3,CX16,PCID,SSE4.1,SSE4.2,MOVBE,POPCNT,AESNI,XSAVE,OSXSAVE,AVX,RDRAND>
  AMD Features=0x28100800<SYSCALL,NX,RDTSCP,LM>
  AMD Features2=0x121<LAHF,ABM,Prefetch>
  Structured Extended Features=0x842421<FSGSBASE,AVX2,INVPCID,NFPUSG,RDSEED,CLFLUSHOPT>
  Structured Extended Features3=0x30000400<MD_CLEAR,L1DFL,ARCH_CAP>
  TSC: P-state invariant
real memory  = 2147418112 (2047 MB)
avail memory = 2044313600 (1949 MB)
```

FWIW we don't use this line but it's a representative input.